app.yaml

```diff
@@ -40,9 +40,17 @@ handlers:
     # WARNING TO DEVELOPERS: Files in this folder may be stale for
     # up to 10 mins after a code release.
     Cache-Control: 'public, max-age=600'
+- url: /assets/(constants\.json)
```

Comment:
When you make the request in a production env, does the request point to /build? I think you can simulate a prod env by passing in --prod_env to start.sh.

(Also, unrelatedly, do you need this additional rule here, given that we have a rule for all of /assets below?)